docs/classification/basic.md

```diff
@@ -66,16 +75,39 @@ plot.feature_importances(est, top_n=5)
 
 A feature importances function is also available in the table module.
 
+Feature Importance refers to techniques that calculate a score for all the input features for a given model the scores simply represent the importance  of each feature:
+
+
+
+
 ```{code-cell} ipython3
 print(table.feature_importances(est))
 ```
 
+A higher score means that the specific feature will have a larger effect on the model that is being used to predict a certain variable, in these case the feature 10 is the most relevant , in order to predict y, wich is the independent variable.
+
+Using sklearn evaluate is a good approach to see the feature importance in a classification task.
+
 Now, let’s see how to generate two of the most common plots for evaluating classifiers: Precision-Recall and ROC.
 
 ```{code-cell} ipython3
 plot.precision_recall(y_true, y_score)
 ```
 
+Precision is one indicator of a machine learning model's performance,the quality of a positive prediction made by the model. 
+Precision refers to the number of true positives divided by the total number of positive predictions between 0 and 1.
```

Comment:
i think you meant recall here. perhaps we can provide a layman's explanation? these concepts are hard to grasp for beginners

Reply:
I write a new explanation from scratch